encord/objects/ontology_labels_impl.py

```diff
@@ -914,19 +914,30 @@ def data_type(self) -> DataType:
 
     @property
     def label_status(self) -> LabelStatus:
+        """
+        Returns current labeling status for the label row.
+
+        **Note**: is not supported for the workflow-based projects. See the documentation about workflows
+        """
+
         if self.__is_tms2_project:
             raise WrongProjectTypeError(
-                '"label"_status property returns incorrect results for TMS2 projects.\
+                '"label"_status property returns incorrect results for workflow-based projects.\
              Please use "workflow_graph_node" property instead.'
             )
 
         return self._label_row_read_only_data.label_status
 
     @property
     def annotation_task_status(self) -> AnnotationTaskStatus:
+        """
+        Returns current annotation task status for the label row.
+
+        **Note**: is not supported for the workflow-based projects. See the documentation about workflows
```

Comment:
Note: This method is not supported for workflow-based projects. Please see our workflow documentation for more details.

Again, if we can hyperlink that would be great. If not, don't worry :)